demangle/vendor/swift/README.md

```diff
@@ -32,7 +32,7 @@ patch is maintained in `1-arguments.patch`.
    5. Build the complete swift project (be very patient, this may take long):
       ```
       $ cd swift
-      $ ./utils/build-script
+      $ ./utils/build-script -S
```

Comment:
fwiw I always prefer long option names in docs or scripts, `--skip-build` explains a lot more then `-S`.